contracts/VaultV3.vy

```diff
@@ -52,6 +52,10 @@ interface IStrategy:
 interface IAccountant:
     def report(strategy: address, gain: uint256, loss: uint256) -> (uint256, uint256): nonpayable
 
+interface ILimitModule:
```

Comment:
Should these be separating into deposit and limit module

Reply:
https://github.com/yearn/yearn-vaults-v3/pull/179/commits/86936020e5b75896744fcf553a131dd5f199aa3a